src/models/categories/selectors.js

```diff
@@ -3,6 +3,8 @@ import { createSelector } from 'reselect';
 
 import { initialState } from './reducer';
 
+export const getCategoryData = ({ id, name, slug, handling_message }) => ({ sub_category: id, name, slug, handling_message });
```

Comment:
Why break the `select...` naming pattern here?

Reply:
This is not a selector, just a service utility, there is in this file `getHasParent` that uses the same naming pattern.